tests/foreman/sys/test_custom_cert_installation.py

```diff
@@ -0,0 +1,120 @@
+"""Test class for ``custom-cert-installation``.
+
+:Requirement: custom-cert-installation
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:CaseComponent: Certificates
+
+:Assignee: okhatavk
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+
+"""
+import pytest
+
+from robottelo.config import settings
+from robottelo.ssh import get_connection
+
+
+@pytest.mark.tier1
+def test_custom_cert_update_satellite(satellite_latest, custom_certs_factory):
+    """Update the satellite instance with new custom certs.
+
+    :id: 6408226e-6c7f-4394-8a60-fb86f058f3fd
+
+    :steps:
+        1. Generate the custom certs
+        2. Update satellite with custom certs
+        3. Assert output does not report SSL certificate error
+        4. Assert all services are running
+
+    :expectedresults: Satellite should be updated with custom certs
+    """
+    satellite = satellite_latest.hostname
+    cert_data = custom_certs_factory(hostname=satellite)
+    with get_connection(hostname=satellite) as connection:
```

Comment:
Please use `satellite_latest.execute()` to send ssh commands to the host. This is taking a brand new broker host instance and then only reading its hostname and using the older ssh methods, which are slower :)